config/manifests/prod/bases/logic-operator-rhel8.clusterserviceversion.yaml

```diff
@@ -289,13 +289,13 @@ spec:
   provider:
     name: Red Hat
   relatedImages:
-  - image: registry.redhat.io/openshift-serverless-1/logic-operator-rhel8:1.34.0
+  - image: registry.redhat.io/openshift-serverless-1/logic-operator-rhel8:1.35.0
     name: IMAGE_LOGIC_SWF_OPERATOR
-  - image: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:1.34.0
+  - image: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:1.35.0
     name: IMAGE_LOGIC_SWF_DEVMODE
-  - image: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:1.34.0
+  - image: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:1.35.0
     name: IMAGE_LOGIC_SWF_BUILDER
   - image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:4564ca3dc5bac80d6faddaf94c817fbbc270698a9399d8a21ee1005d85ceda56
     name: IMAGE_KUBE_RBAC_PROXY
-  replaces: logic-operator-rhel8.v1.33.0
+  replaces: logic-operator-rhel8.v1.34.0
   version: 0.0.0
```

Comment:
Shouldn't be there version updated as well? Here is version 0.0.0 is this correct for this csv file